examples/v1beta1/pipelineruns/conditional-pipelinerun-with-same-condition-refer.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Condition
+metadata:
+  name: is-equal
+spec:
+  params:
+  - name: left
+    type: string
+  - name: right
+    type: string
+  check:
+    image: alpine
+    script: |
+      #!/bin/sh
+      if [ $(params.left) = $(params.right) ]; then
+        echo "$(params.left) == $(params.right)"
+        exit 0
+      else
+        echo "$(params.left) != $(params.right)"
+        exit 1
+      fi
+---
+apiVersion: tekton.dev/v1alpha1
```

Comment:
For the v1beta1 folder, I think the apiVersions for Pipeline/Task/Runs need to be v1beta1